### PR TITLE
remove jsmin dependency

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**
jsmin is a unmaintained since a while and breaks the build

im under the impression we practically only use it to remove some white-space from js code to send to selenium, so we should be fine without it

### CODE REVIEW

**Comment by quarckster** [comment]
LGTM

**Comment by rsnyman** [comment]
LGTM

File: src/widgetastic/browser.py [hunk 1]
```diff
@@ -657,11 +672,11 @@ def move_by_offset(self, x: int, y: int) -> None:
         ActionChains(self.selenium).move_by_offset(x, y).perform()
 
     @retry_stale_element
-    def execute_script(self, script: str, *args, **kwargs) -> Any:
+    def execute_script(self, script: str, *args, silent=False, **kwargs) -> Any:
         """Executes a script."""
         from .widget import Widget
 
-        if not kwargs.pop("silent", False):
```
Comment (mshriver) [inline src/widgetastic/browser.py:664]: This was also removing `silent` from the `kwargs` which are passed to `selenium.execute_script`.   Doesn't look like this function signature takes kwargs, maybe this isn't covered by unit tests right now?

**Review by mshriver — COMMENTED** [review]
Fine with removing jsmin dependency - thanks for cleaning that up!

There is a logical change here too with handling of `kwargs` in `execute_script` that I have some concern about.

**Comment by RonnyPfannschmidt** [comment]
I'll split those 2 changes and send a additional pr

**Comment by mshriver** [comment]
@RonnyPfannschmidt fine with it being in the same PR, just want to make sure unit tests are covering the call stack testing the execution of the script.

File: src/widgetastic/browser.py (hunk 1 above)
Comment (mshriver) [reply src/widgetastic/browser.py:664]: I took another look - `kwargs` is passed to `self.element()`, not execute_script. This isn't an issue.

**Review by mshriver — APPROVED** [review]

**Comment by mshriver** [comment]
Small rebase and I'm happy to merge @RonnyPfannschmidt

**Pull request merged**